downstream/assemblies/platform/assembly-deploying-chatbot-operator.adoc

```diff
@@ -0,0 +1,77 @@
+For more information about nesting assemblies, see: https://redhat-documentation.github.io/modular-docs/#nesting-assemblies
+See also the complementary step on the last line of this file.
+////
+
+ifdef::context[:parent-context: {context}]
+
+
+[id="deploying-chatbot-operator"]
+= Deploying the {AAPchatbot} on {OCPShort}
+
+:context: deploying-chatbot-operator
+
+[role="_abstract"]
+
+The {AAPchatbot} is an intuitive chat interface embedded within the {PlatformNameShort}, utilizing generative artificial intelligence (AI) to answer questions about the {PlatformNameShort}. 
+
+The chat experience in the {AAPchatbot} interacts with users in their natural language prompts in English, and uses large language models (LLMs) to generate quick, accurate, and personalized responses. These responses empower Ansible users to work more efficiently, thereby improving productivity and the overall quality of their work. 
+
+{AAPchatbot} requires the following configurations:
+
+* Installation of {PlatformNameShort} {PlatformVers} on {OCPShort}
+* Deployment of an LLM served by either Red Hat Enterprise Linux® AI or Red Hat OpenShift AI
+
+[NOTE]
+====
+Red Hat does not collect any telemetry data from user interactions with the {AAPchatbot}. 
+====
+
+== Accessing the {AAPchatbot}
+
+Perform the following tasks to set up and use the {AAPchatbot} in your {PlatformNameShort} instance on the {OCPShort} environment:
+
+[%header,cols="35%,65%"]
+|====
+| Task 
+| Description
+
+|Deploy the {AAPchatbot} on {OCPShort}
+a|An {PlatformNameShort} administrator who wants to deploy the {AAPchatbot} for all Ansible users so that they can get answers to their questions about the {PlatformNameShort}.
+
+Perform the following tasks:
+
+. Create a chatbot configuration secret.
+. Reconfigure the {PlatformNameShort} to use the chatbot connection secret. 
+
+| Access and use the {AAPchatbot}
+| All Ansible users belonging to an organization that has {AAPchatbot} configured who want to use the intelligent assistant to answer questions about the {PlatformNameShort}.
+|====
+
+== Prerequisites
+
+*For the {PlatformNameShort} {PlatformVers}*: 
+
+* Your system meets the minimum system requirements for installing the {OperatorPlatformNameShort} and the required infrastructure topology. For more information, see link:{URLTopologies}/ocp-topologies[Operator topologies] in _{TitleTopologies}_.
+* You have installed {PlatformNameShort} {PlatformVers} on your {OCPShort} environment. 
+* You have administrator privileges for the {PlatformNameShort}.
+* You have provisioned an OpenShift cluster with Operator Lifecycle Management installed.
+
+*For the Large Language Model (LLM)*:
+
+* You must have configured a Large Language Model (LLM) provider that you will use before deploying the {AAPchatbot}. You can use an LLM served by either Red Hat Enterprise Linux® AI or Red Hat OpenShift AI.
++
+For more information, refer to the following resources:
```

Comment:
This is a nit, but suggest changing "refer to" -> "see" as it looks like that's our standard.

Reply:
Made this change, thank you